ocaml/xapi/xapi_message.ml

```diff
@@ -817,10 +817,10 @@ let handler (req : Http.Request.t) fd _ =
           (ExnHelper.string_of_exn e)
   )
 
-(* Export messages and send to another host/pool over http. *)
-let send_messages ~__context ~cls ~obj_uuid ~session_id ~remote_address =
-  let msgs = get ~__context ~cls ~obj_uuid ~since:(Date.of_float 0.0) in
-  let body = export_xml msgs in
+(* Export and send host messages to another host/pool over http. *)
```

Comment:
minor nitpick:

are they host messages, or messages for an object (VM) that were stored on that host? Given that this is only invoked by VM migration and the 'cls' parameter is unused here I'd say 'export and send messages stored on the current host' (they are not necessarily messages about the host).

Also if 'cls' really is unused now then you can drop that parameter: the message class is implied by the list of messages you pass.

Reply:
good point, actually. I'll update the comment.

The `cls` is actually used, that was a mistake on my part